main.Rmd

```diff
@@ -313,12 +313,12 @@ To reproduce this dataset, the subsequent steps are required. They have been tes
  
 5.  Navigate to the project directory and open the file *macro_mohp_feature.Rproj* with RStudio.
 
-6.  Install the package *renv* by running following command in the console
+6.  Install the package *renv* by running following command in the R-console
 ```{r, eval=FALSE}
 install.packages("renv")
 ```
  
-7.  Install all package dependencies with the subsequent line (Note that under Linux and MacOS some R-packages have system dependencies, such as the package _sf_, which depends on `libgeos-dev`, among others. Please consult the respective documentation when facing an issue.)
+7.  Install all package dependencies with the subsequent line in the R-console. Note that under Linux and MacOS some R-packages have system dependencies, such as the package `sf`, which depends on `libgeos-dev`, among others. Please consult the respective documentation when facing an issue.
```

Comment:
`libgeos-dev` is the name of the dependency for Debian and might be called differently for MacOS or other Linux distributions.